mongox/get.go

```diff
@@ -10,24 +10,19 @@ import (
 	"github.com/w6d-io/x/logx"
 )
 
-func (m *MongoDB) Get(filter interface{}, data interface{}) error {
+func (m *MongoDB) Get(filter interface{}) (CursorAPI, error) {
```

Comment:
Not sure I'd like to change this signature as is match the others
Why did you do that ?